repo/repo.go

```diff
@@ -29,9 +29,15 @@ func CurrentIdentity() (*Identity, error) {
 		return nil, err
 	}
 
+	signingKey, err := GetConfig("user.signingkey")
+//         if err != nil {
```

Comment:
Ah I see, in that case we should probably catch the exit code and not err if it's 1. Might need to modify the implementation of GetConfig for that